distribution/src/main/release/samples/pom.xml

```diff
@@ -210,6 +216,49 @@
             </plugins>
         </pluginManagement>
     </build>
+    <repositories>
+        <repository>
+            <id>spring.snapshots</id>
+            <url>https://repo.spring.io/snapshot/</url>
+            <name>Spring Snapshot Repo</name>
+            <snapshots>
+                <enabled>true</enabled>
+            </snapshots>
+            <releases>
+                <enabled>false</enabled>
+            </releases>
+        </repository>
+        <repository>
+            <id>spring.milestone</id>
+            <url>https://repo.spring.io/milestone/</url>
+            <name>Spring Milestone Repo</name>
+        </repository>
+        <repository>
+            <id>netflix.candidates</id>
+            <url>https://artifactory-oss.prod.netflix.net/artifactory/maven-oss-candidates</url>
+            <snapshots>
+                <enabled>false</enabled>
+            </snapshots>
+        </repository>
+    </repositories>
+    <pluginRepositories>
+        <pluginRepository>
```

Comment:
Same here, I think we could get rid of snapshots repo

Reply:
Done.